README.md

```diff
@@ -20,21 +20,20 @@ The library is modular so you can use as little or as much as you need.
 The fully compiled, minified library is available via CDN for inclusion in your application.
 
 ```
-<link href="//unpkg.com/fundamental-styles@latest/dist/fundamental-styles.min.css" rel="stylesheet">
+<link href="//unpkg.com/fundamental-styles@latest/dist/fundamental-styles.css" rel="stylesheet">
 ```
 
 ### NPM Package
 
-The compiled CSS for the full library and modules, e.g., core, layout, etc., are distributed via [NPM](https://www.npmjs.com/package/fundamental-styles) along with the SASS source.
+The compiled CSS for the full library and modules, e.g., core, layout, etc., are distributed via [NPM](https://www.npmjs.com/package/fundamental-styles).
 
 This gives you incredible flexibility to use individual components and enable [advanced customization](https://github.com/SAP/fundamental-styles/wiki/Advanced-Customization) options.
 
```

Comment:
url does not exist, and perhap neither does the concept.

Reply:
Deleted.

Reply:
actually, i meant this line

> This gives you incredible flexibility to use individual components and enable [advanced customization](https://github.com/SAP/fundamental-styles/wiki/Advanced-Customization) options.

Reply:
Yeah sorry I meant deleted locally, I hadn't pushed it yet :)